deployment/docker-compose.yml

```diff
@@ -79,6 +79,8 @@ services:
     container_name: "geosight_db"
     volumes:
       - database:/var/lib/postgresql/13
+    ports:
```

Comment:
This is just because I'm using pycharm without devcontainers, so I was starting the database locally and connecting to it using `postgis://...localhost:5432...`. Not sure if anybody else on the team has experience developing GeoSight without VSCode?